plugins/global-search/src/components/SearchScene.tsx

```diff
@@ -31,10 +31,8 @@ export function SearchScene() {
     }, [])
 
     useEffect(() => {
-        void framer.showUI(
-            getPluginUiOptions({ query: deferredQuery, hasResults, hasFinalResults: !isIndexing && !isFiltering })
-        )
-    }, [deferredQuery, hasResults, isIndexing, isFiltering])
+        void framer.showUI(getPluginUiOptions({ query: deferredQuery, hasResults, hasFinalResults: !isIndexing }))
```

Comment:
Nit: I trip over `hasFinalResults` a little bit, as if it's talking about different final results, which aren't the same as plain results. Maybe `areResultsFinal`?